src/components/pages/profile-overview/profile-overview.tsx

```diff
@@ -188,9 +188,11 @@ const UserAddresses: React.FC<{
         href={`/account/profile/addresses/${primaryAddress.id}`}
       >
         <CategoryContent>
-          <CategoryName>
-            {capitalizeFirstLetter(primaryAddress.kind)}
-          </CategoryName>
+          {primaryAddress.kind && (
+            <CategoryName>
+              {capitalizeFirstLetter(primaryAddress.kind)}
+            </CategoryName>
+          )}
```

Comment:
This is mandatory because we can create addresses directly with Profile calls and if the payload contains a `null` `kind` field it crashes.